src/devices/Gui/Interop/Interop.uinput.cs

```diff
@@ -0,0 +1,205 @@
+
+        internal const int EV_SYN = 0x00;
+        internal const int EV_KEY = 0x01;
+        internal const int EV_REL = 0x02;
+        internal const int EV_ABS = 0x03;
+        internal const int EV_MSC = 0x04;
+        internal const int EV_SW = 0x05;
+        internal const int EV_LED = 0x11;
+        internal const int EV_SND = 0x12;
+        internal const int EV_REP = 0x14;
+        internal const int EV_FF = 0x15;
+        internal const int EV_PWR = 0x16;
+        internal const int EV_FF_STATUS = 0x17;
+        internal const int EV_MAX = 0x1f;
+        internal const int EV_CNT = (EV_MAX + 1);
+
+        internal const int BTN_MOUSE = 0x110;
+        internal const int BTN_LEFT = 0x110;
+        internal const int BTN_RIGHT = 0x111;
+        internal const int BTN_MIDDLE = 0x112;
+        internal const int BTN_SIDE = 0x113;
+        internal const int BTN_EXTRA = 0x114;
+        internal const int BTN_FORWARD = 0x115;
+        internal const int BTN_BACK = 0x116;
+        internal const int BTN_TASK = 0x117;
+
+        /*
+        * Relative axes
+        */
+
+        internal const int REL_X = 0x00;
+        internal const int REL_Y = 0x01;
+        internal const int REL_Z = 0x02;
+        internal const int REL_RX = 0x03;
+        internal const int REL_RY = 0x04;
+        internal const int REL_RZ = 0x05;
+        internal const int REL_HWHEEL = 0x06;
+        internal const int REL_DIAL = 0x07;
+        internal const int REL_WHEEL = 0x08;
+        internal const int REL_MISC = 0x09;
+
+        /*
+        * Absolute axes
+        */
+
+        internal const int ABS_X = 0x00;
+        internal const int ABS_Y = 0x01;
+        internal const int ABS_Z = 0x02;
+        internal const int ABS_RX = 0x03;
+        internal const int ABS_RY = 0x04;
+        internal const int ABS_RZ = 0x05;
+        internal const int ABS_THROTTLE = 0x06;
+        internal const int ABS_RUDDER = 0x07;
+        internal const int ABS_WHEEL = 0x08;
+        internal const int ABS_GAS = 0x09;
+        internal const int ABS_BRAKE = 0x0a;
+        internal const int ABS_HAT0X = 0x10;
+        internal const int ABS_HAT0Y = 0x11;
+        internal const int ABS_HAT1X = 0x12;
+        internal const int ABS_HAT1Y = 0x13;
+        internal const int ABS_HAT2X = 0x14;
+        internal const int ABS_HAT2Y = 0x15;
+        internal const int ABS_HAT3X = 0x16;
+        internal const int ABS_HAT3Y = 0x17;
+        internal const int ABS_PRESSURE = 0x18;
+        internal const int ABS_DISTANCE = 0x19;
+        internal const int ABS_TILT_X = 0x1a;
+        internal const int ABS_TILT_Y = 0x1b;
+        internal const int ABS_TOOL_WIDTH = 0x1c;
+        internal const int ABS_VOLUME = 0x20;
+        internal const int ABS_MISC = 0x28;
+
+        internal const int SYN_REPORT = 0;
+
+        [StructLayout(LayoutKind.Sequential)]
+        internal struct input_id
+        {
+            public UInt16 bustype;
```

Comment:
similar as a previous question, any reason in this file to use the UInt64, UInt16, etc rather than the ulong, ushort, etc versions?